packages/regl-worldview/src/stories/cameraState.stories.js

```diff
@@ -148,7 +148,7 @@ class CameraStateStory extends React.Component<Props, State> {
   }
 }
 
-const stories = storiesOf("Worldview", module);
+const stories = storiesOf("Worldview - cameraState", module);
```

Comment:
Looks like you can use `/` for nesting: https://storybook.js.org/docs/basics/writing-stories/#nesting-stories